lib/model/QubitObject.php

```diff
@@ -305,6 +305,35 @@ public function getStatus($options = [])
         return QubitStatus::getOne($criteria);
     }
 
+    /**
+     * Check to see if note is saved in memory (field was updated).
+     *
+     * Used by ISAD, DACS, and RAD templates when duplicating a description,
+     * specifcally for updating the 'Language and script notes' field.
+     *
+     * @return array $notes
+     */
+    public function getMemoryNotesByType(array $options = [])
+    {
+        // Check memory for notes
+        if (!count($this->notes) < 0) {
+            return $this->getNotesByType($options);
+        }
+
+        $noteTypeId = $options['noteTypeId'];
+        $notes = new ArrayObject();
+
+        foreach ($this->notes as $note) {
+            if ($note->typeId == $noteTypeId) {
```

Comment:
I'm guessing these will always be string? If so we should prefer `===` since we're sure they're the same data type.

Reply:
this is actually an integar (`$note->typeId`) and a string (`$noteTypeId`) so i kept it as `==`